index.js

```diff
@@ -16,6 +16,7 @@
 var EventEmitter = require('events').EventEmitter;
 var debug = require('debug')('koa-redis');
 var redis = require('redis');
+var ioredis = require("ioredis");
```

Comment:
I'd like to switch to ioredis and remove redis, then we can release a major version.

Reply:
I'll take care of this now

Reply:
Done